recipes/vega/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.3.0" %}
+
+package:
+  name: vega
+  version: {{ version }}
+
+source:
+  fn: vega-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/v/vega/vega-{{ version }}.tar.gz
+  md5: 59c915a18179ec8de48798a7713e7ee5
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - jupyter
+
+  run:
+    - python
+    - ipython
+    - jupyter
+
+test:
+  imports:
+    - vega
+
+about:
+  home: https://github.com/vega/ipyvega
+  license: BSD-3-Clause
+  summary: 'An IPython/Jupyter widget for Vega and Vega-Lite'
+
+extra:
+  recipe-maintainers:
+    - jakevdp
+    - ellisonbg
+    - domoritz
```

Comment:
Yep, sounds good!

On Fri, Jun 3, 2016 at 10:06 AM, jakirkham <notifications@github.com> wrote:

> In recipes/vega/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/704#discussion_r65741998>
> :
>
> > +    - jupyter
> > +
> > +test:
> > +  imports:
> > +    - vega
> > +
> > +about:
> > +  home: https://github.com/vega/ipyvega
> > +  license: BSD-3-Clause
> > +  summary: 'An IPython/Jupyter widget for Vega and Vega-Lite'
> > +
> > +extra:
> > +  recipe-maintainers:
> > +    - jakevdp
> > +    - ellisonbg
> > +    - domoritz
>
> Are both of you ok with being added as maintainers, @ellisonbg
> <https://github.com/ellisonbg> and @domoritz <https://github.com/domoritz>
> ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/704/files/65867c991a31d1a913920eead6f756c2a580e3c8#r65741998>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABr0PcSNVgtWKnUM71xO85MFQrnokfAks5qIF8FgaJpZM4ItCx3>
> .
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com